utils/s3_file_util.py

```diff
@@ -0,0 +1,71 @@
+import boto3, uuid, io, os
+from PIL         import Image
+
+from django.conf import settings
+
+class S3FileUtils:
+    s3 = boto3.resource(
+        's3',
+        aws_access_key_id     = settings.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key = settings.AWS_SECRET_ACCESS_KEY
+    )
+    bucket = s3.Bucket(settings.AWS_STORAGE_BUCKET_NAME)
```

Comment:
s3 class 만드신거 정말 잘 하셨습니다. 
1가지만, 제가 조언을 드리자면, 지금 현재 클래스는 하나의 버킷에 대해서만 작동하게 구현 되어 있습니다. 그래서 활용도가 떨어집니다. 그래서 다음과 같이 수정하시면 여러 버킷을 사용해야 할 경우 활용도가 증가합니다. 
아래 내용은 지금 당장 수정 사항은 아니니 참고만 해주세요.
```suggestion
    def __init__(self, S3_ACCESS_KEY_ID, S3_SECRET_ACCESS_KEY, AWS_STORAGE_BUCKET_NAME): 
        s3 = boto3.resource(
            's3',
            aws_access_key_id     = AWS_ACCESS_KEY_ID,
            aws_secret_access_key = AWS_SECRET_ACCESS_KEY
        )
        bucket = self.s3.Bucket(AWS_STORAGE_BUCKET_NAME)
```